projects/vdk-plugins/vdk-ipython/tests/test_plugin.py

```diff
@@ -111,11 +123,11 @@ async def test_extension_with_ingestion_job(ip, tmpdir):
     )
 
     # get the data that is going to be ingested
-    ip.get_ipython().run_cell("import requests")
+    ip.get_ipython().run_cell("import json")
     ip.get_ipython().run_cell(
-        'response = requests.get("https://jsonplaceholder.typicode.com/todos/1")'
+        """raw = '{ "userId": 1, "id": 1, "title": "delectus aut autem", "completed": false }' """
```

Comment:
Nit: Might be a good idea to add a comment explaining what this represents (placeholder).